cardano-node/src/Cardano/Tracing/Shutdown.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# OPTIONS_GHC -Wno-orphans  #-}
+module Cardano.Tracing.Shutdown where
+
+import Prelude (IO)
+import Data.Text (Text)
+
+import Cardano.Logging (LogFormatting (..))
+import Cardano.Node.Handlers.Shutdown
+import Cardano.Node.Tracing.Compat
+import Cardano.Node.Tracing.Tracers.Shutdown
+import Cardano.BM.Tracing (HasPrivacyAnnotation (..),
+         HasSeverityAnnotation (..), Severity (..), ToObject (..),
+         Transformable (..))
+import Cardano.BM.Data.Tracer (HasTextFormatter (..),
+         trStructuredText)
+
+instance HasPrivacyAnnotation  ShutdownTrace
```

Comment:
Why can't these live in `Cardano.Tracing.Tracers`?

Reply:
Cardano.Tracing.Tracers constructs the tracers, but all the instances are in seperate modules, I would keep with this.

Reply:
Apologies, these should live with the definition `data ShutdownTrace`. The other instances live separately in modules because they are instances of types that don't exist in `cardano-node`.

Reply:
@Jimbo4350, the idea is that the entire `Cardano.Tracing` can be removed painlessly in the future -- so it's separate, and not mixed with the other code.

What do you think?

Reply:
Right, understood :+1: